master/pkg/agent/exit.go

```diff
@@ -14,7 +14,7 @@ type ContainerFailure struct {
 }
 
 func (c ContainerFailure) Error() string {
-	return fmt.Sprintf("%s: %s", c.FailureType, c.ErrMsg)
+	return fmt.Sprintf("%s: %s %d", c.FailureType, c.ErrMsg, *c.ExitCode)
```

Comment:
non-blocking: Add in a touch of formatting:
```suggestion
	return fmt.Sprintf("%s: %s (exit code %d)", c.FailureType, c.ErrMsg, *c.ExitCode)
```

Reply:
done